src/app/routes/main/routes/data/routes/recipe/recipe.component.html

```diff
@@ -136,19 +136,21 @@ <h3>{{ obj.name }}</h3>
       <tr>
         <th>{{ 'data.unlockedBy' | translate }}</th>
         <td>
-          <a
-            [routerLink]="'/data/technologies/' + obj.unlockedBy"
-            queryParamsHandling="preserve"
-          >
-            <button
-              pButton
-              pRipple
-              type="button"
-              class="p-button-text"
-              [label]="data().itemEntities[obj.unlockedBy].name"
-              [icon]="obj.unlockedBy | iconSmClass: 'recipe'"
-            ></button>
-          </a>
+          @for (unlock of asArray(obj.unlockedBy); track unlock) {
```

Comment:
Never used angular before but I think this should be correct. Maybe the track is unnecessary?

Reply:
Track is always necessary, but with primitive values it's fine to track the value or `$index`